source/releasenotes/changes.rst

```diff
@@ -13,14 +13,542 @@
    specific language governing permissions and limitations
    under the License.
 
+Issues Fixed in 4.13.1.0
+========================
 
 
-Changes in |release| since 4.12
-===============================
+Issues Fixed in 4.13.1.0
+------------------------
```

Comment:
why the double header?

Reply:
Due yo headers in the ToC table on yhe left. Was weird for me as well, but needed as such.